src/common/features/component/Item.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from "./Item.module.scss";
+
+export function Item(props) {
+    //can add className to this list, the div item might be div className
+    var {item,className,...props} = props;
+    return (
+        
+        <div className={classNames(styles.Item, item)}>
```

Comment:
I see you're passing the the whole item component as a scss class. Should this be the className prop?